### DISCUSSION TITLE
Tagging entries

### DISCUSSION THREAD

**Discussion by treequell** [keepassxreboot/keepassxc#6254] [Ideas]
I have recently started using KeePassXC. Thank you for all your work on this software.

I was hoping to be able to tag entries in my .kdbx database, but I can't find any way to do so in KeepassXC. I have looked through the KeePassXC documentation (User Guide, Wiki, Blog, Github Issues, Pull Requests and Discussions) but I could not find any mention of such a feature anywhere.

I am able to tag entries using KeePass and KeeWeb. I would be interested to hear what others think about having the same feature in KeePassXC.

In the meantime, I will be filing entries in folders but I generally prefer tagging when working with databases, and I would be happy to see this feature in KeepassXC.

**Comment by droidmonkey** [comment]
Pending feature, aiming for 2.7.0

**Reply by mackayem** [reply]
Really happy to hear that. Excited to have that feature as well. Thanks for all your hard work!
